src/Razor/test/Microsoft.CodeAnalysis.Razor.Workspaces.Test/ProjectSystem/DefaultDocumentSnapshotTest.cs

```diff
@@ -80,20 +79,20 @@ public async Task GCCollect_OutputIsNoLongerCached()
 
             // Assert
             Assert.False(LegacyDocument.TryGetGeneratedOutput(out _));
-            Assert.False(LegacyDocument.TryGetGeneratedOutputVersionAsync(out _));
+            Assert.False(LegacyDocument.TryGetGeneratedCSharpOutputVersionAsync(out _));
```

Comment:
Does `LegacyDocument.GetGeneratedOutputAsync` not create the HTML document? Wondering if we should expand tests in this file to include the Html route